packages/next/build/webpack/plugins/webpack-conformance-plugin/constants.ts

```diff
@@ -0,0 +1,6 @@
+import { red, yellow } from 'kleur'
```

Comment:
This will fail Yarn's PnP so there's no dependency on this package. Can you please switch to `chalk`?

Reply:
done